### ISSUE TITLE
Missing headers in log request

### ISSUE THREAD

**Issue by flaper87** [curiefense/curiefense#410]
**Describe the bug**
Standard headers seem to be missing in the data sent to curielogger

This is what curielogger receives:

```
...
"headers":{"x-forwarded-for":"52.5.112.215","x-envoy-
external-address":"52.5.112.215","x-request-id":"ea7194ea-85ef-4a3f-b34f-dfde9f3e2753","user-agent":"Reblaze Browser 5.4.3.2","x-forwarded-proto":"http","accept":"*/*"}
...
```

And the headers tag is null:

`headers:null`

**To Reproduce**
1. docker-compose up 

Expected result:

We should have all the base headers like Accept, Path, Host, Method, etc.

If these are standard fields, we can also put them in the request attributes, which already has a structure:

https://github.com/curiefense/curiefense/blob/f0265542cb2679920e0b93b65cf861efe7867016/curiefense/curielogger/pkg/entities/misc.go#L61-L68

**Comment by tzuryby** [comment]
A slight correction for this one.

Host header in http2 is privided as `:authority` and so are method and path and perhaps other _pseudo_ headers.
The rust code should store those values under attributes.

The lua provide this to the rust under `meta` (see code in https://github.com/curiefense/curiefense/blob/main/curiefense/curieproxy/lua/session_envoy.lua#L41)

and this should be retrived back under attributes.

@phil777 -- this is regarding our call today.

**Comment by bartavelle** [comment]
This should be fixed in #442, however it also adds the `authority` header, and potentially other, unknown, headers (it sends back what was in the meta map) :(

**Comment by flaper87** [comment]
> This should be fixed in #442, however it also adds the `authority` header, and potentially other, unknown, headers (it sends back what was in the meta map) :(

Unknown headers will be ignored. Let me know If we should add them to the mapping

**Comment by bartavelle** [comment]
Should be ok now.
